.github/workflows/ci.yaml

```diff
@@ -14,21 +14,25 @@ jobs:
   build:
     env:
       COVER: true
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-latest
     steps:
 
     - name: Check out code
       uses: actions/checkout@v4
 
-    - name: Set up Go 1.21
+    - name: Set up Go
       uses: actions/setup-go@v5
       with:
-        go-version: 1.21
+        # renovate: datasource=golang-version depName=golang
+        go-version: 1.21.6
```

Comment:
Makes it more repeatable. I feel like Golang doesn't create new patch versions too often.

Reply:
I would actually just prefer to always use the latest version. Instead of having to relying on renovate to update the workflows

Reply:
Just set up some workflows using this action as well and it has a go-version-file input now. I'll use that instead.